tools/cdk-build-tools/bin/cdk-test.ts

```diff
@@ -31,7 +37,14 @@ async function main() {
   }
 
   const testFiles = await unitTestFiles();
-  if (testFiles.length > 0) {
+  const useJest = 'jest' in currentPackageJson();
+
+  if (useJest) {
+    if (testFiles.length > 0) {
+      throw new Error(`Jest is enabled, but ${testFiles.length} nodeunit tests were found!`);
```

Comment:
So a package must be entirely jest or nodeunit to work?

Reply:
Yes. We *could* allow both but would have to merge coverage data & the likes, which is possible but ugly as heck. I don't need this right now so not getting into this... We can do that later if there is a real value to it.